tests/commerce_integration/TEC/Tickets/Commerce/Gateways/Stripe/Hooks_Test.php

```diff
@@ -178,34 +178,34 @@ public function test_it_should_bail_async_stripe_webhooks_when_end_result_is_don
 	}
 
 	public function checkout_request_data_provider(): Generator {
-		yield 'empty request' => [
-			function() {
-				$post = self::factory()->post->create(
-					[
-						'post_type' => 'page',
-					]
-				);
-				$ticket_id_1 = $this->create_tc_ticket( $post, 10 );
-				$order = $this->create_order_through_stripe( [ $ticket_id_1 => 1 ], [ 'order_status' => Pending::SLUG ] );
-
-				return [ $order->ID, false ];
-			},
-		];
-
-		yield 'with request params - no pi' => [
-			function() {
-				$post = self::factory()->post->create(
-					[
-						'post_type' => 'page',
-					]
-				);
-				$ticket_id_1 = $this->create_tc_ticket( $post, 10 );
-				$order = $this->create_order_through_stripe( [ $ticket_id_1 => 1 ], [ 'order_status' => Pending::SLUG ] );
-				$_REQUEST['payment_intent'] = 'pi_123';
-				$_REQUEST['payment_intent_client_secret'] = 'pi_123_secret';
-				return [ $order->ID, false ];
-			},
-		];
+		// yield 'empty request' => [
```

Comment:
Should we just remove these cases? Or is there a reason to leave them commented out?